### ISSUE TITLE
bug: Find Members button not navigating properly

### ISSUE THREAD

**Issue by techno-disaster** [bartekpacia/mentorship-client#113]
**Describe the bug**
When the user is not in a relation we show a find members button on the relation page, but this is not working as intended, the button does not navigate in the bottom navy bar.

**To Reproduce**
1. Go to 'Relation page', if you are not in a relation you will see a find members button, incase you are just use this account ```qwerty12:12345678``` once you tap on the find members button you will be able to see the members page for the first time but the bottom navy bar will still say Relation page

**Comment by techno-disaster** [comment]
@bharat-1809 would you like to work on this? or anyone else from @anitab-org/coding-team ?

**Comment by HaripriyaB** [comment]
Can I take this up?

**Comment by Akanksha1212** [comment]
Sure @HaripriyaB

**Comment by techno-disaster** [comment]
@HaripriyaB whats the status of the issue?

**Comment by HaripriyaB** [comment]
Working on it. But I'm stuck at how to navigate from one tab to another with the button click. I tried to navigate but it threw error.

**Comment by Akanksha1212** [comment]
Hi, @HaripriyaB Were you able to resolve the error? If not can you share the details about the error here

**Comment by HaripriyaB** [comment]
When I add navigator in onpressed of find members button it shows error.

**Comment by techno-disaster** [comment]
Yep, you will have to change the whole navigation for bottom navy bar probably. Use bloc like we did before. Check how tabs were navigated before #108 commit. @HaripriyaB Once you get this it will be super easy

**Comment by techno-disaster** [comment]
@HaripriyaB if you are still stuck with this, I can take over this issue?

**Comment by HaripriyaB** [comment]
yeah ..I guess I'm not able to figure out the solution.I tried all possible ways. So you take it up.

**Comment by bharat-1809** [comment]
@HaripriyaB and @Techno-Disaster Can I take up this?

**Comment by techno-disaster** [comment]
Hey @bharat-1809 i already fixed this and will be sending an PR tonight can you take up one of the other issues?

**Comment by bharat-1809** [comment]
@Techno-Disaster Yeah, Sure :)

**Comment by isabelcosta** [comment]
@HaripriyaB please take a look at #119 PR so you can see what you were possibly missing from your solution :)

**Comment by darahask** [comment]
Can I solve this issue please, Thankyou

**Comment by techno-disaster** [comment]
> Can I solve this issue please, Thankyou

Hi this is fixed in #119

**Comment by darahask** [comment]
But I recently forked the develop branch of the project, it has two issues 
1. It works only for the first time.
2. It doesn't change the tab focus.
so I tried to solve it.

**Comment by darahask** [comment]
Understood related Pr hasn't been merged yet, Thank you.
Any other feature or issue I can work upon @Techno-Disaster.
